src/execution/execute.ts

```diff
@@ -1469,7 +1467,11 @@ export const defaultFieldResolver: GraphQLFieldResolver<unknown, unknown> =
  * If the operation succeeded, the promise resolves to an AsyncIterator, which
  * yields a stream of ExecutionResults representing the response stream.
  *
- * This function does not support incremental delivery (`@defer` and `@stream`).
+ * This function also supports experimental incremental delivery directives
+ * (`@defer` and `@stream`). To use these directives, they should be added to
+ * the schema and TS generic parameter TMaybeIncremental should be set to `true`
+ * (default: false).
+ * * This function does not support incremental delivery (`@defer` and `@stream`).
```

Comment:
It looks like an artifact from https://github.com/graphql/graphql-js/pull/3732
@yaacovCR Can you please remove it?
Otherwise, it's ready to be merged 👍

Reply:
1d34ccde